nexus-repository-p2/src/main/java/org/sonatype/nexus/repository/p2/internal/search/P2SearchMappings.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.p2.internal.search;
+
+import java.util.List;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.sonatype.goodies.common.ComponentSupport;
+import org.sonatype.nexus.repository.p2.internal.P2Format;
+import org.sonatype.nexus.repository.rest.SearchMapping;
+import org.sonatype.nexus.repository.rest.SearchMappings;
+
+import com.google.common.collect.ImmutableList;
+/**
+ * @since 0.next
```

Comment:
Q: I know this is merged but is this established behavior in our plugins?  Also I feel like when we've been going live it's been 1.0.0 (see R), which would make 1.next make more sense to me.  But as long as it's established/consistent, I don't care, just curious.

Reply:
Well, currently in pom file is 0.0.4 version, so I think we should change it to 1.next in scope of release ticket in all project?

Reply:
I think it's a good idea as it represents a big transition (plugin to product) however I am not sure if we have a standard...this may be establishing one=)  Thoughts @jlstephens89 ?  Or maybe also Anatoliy or Sergii (don't have their IDs handy, am just reviewing from home on vacation).